Source/cmake/WebKitFeatures.cmake

```diff
@@ -77,15 +77,15 @@ macro(WEBKIT_OPTION_BEGIN)
     set(_SETTING_WEBKIT_OPTIONS TRUE)
 
     if (WTF_OS_WINDOWS)
```

Comment:
This code was added by me to disable JIT for Windows.
https://github.com/WebKit/WebKit/pull/28269
Can we just remove this code?